### DISCUSSION TITLE
Support for EF Projections

### DISCUSSION THREAD

**Discussion by dde2801** [Eventuous/eventuous#152] [Ideas]
Do you plan to add support soon for Projections to Read Models in EF (SQL Server) or if supported, could you please provide some guidance on how to implement it?

**Comment by alexeyzimarev** [comment]
I am not sure what needs to be done. Checkpoint stores for SQL Server and PostgreSQL are available. Maybe it would make sense to add one using EF so it can use other providers. For the rest, it's just a subscription with projecting handlers. Inject the DbContext to the handler and do whatever you want.

However, I am not sure if EF projections are that useful. They might play well with https://github.com/Eventuous/eventuous/issues/159 if it ever gets done, but in other cases, EF works as load-change-update, and it's just too slow.
